src/app/clusters/ota-requestor/GenericOTARequestorDriver.cpp

```diff
@@ -64,18 +71,20 @@ void GenericOTARequestorDriver::Init(OTARequestorInterface * requestor, OTAImage
 
     if (mImageProcessor->IsFirstImageRun())
     {
-        SystemLayer().ScheduleLambda([this] {
-            CHIP_ERROR error = mImageProcessor->ConfirmCurrentImage();
-
-            if (error != CHIP_NO_ERROR)
-            {
-                ChipLogError(SoftwareUpdate, "Failed to confirm image: %" CHIP_ERROR_FORMAT, error.Format());
-                mRequestor->Reset();
-                return;
-            }
-
-            mRequestor->NotifyUpdateApplied();
-        });
+        ScheduleDelayedAction(
+            mDelayNotifyUpdateAppliedSeconds,
```

Comment:
Looks like mDelayNotifyUpdateAppliedSeconds is used to delay confirming current image, which in turns causes NotifyUpdateApplied() to be delayed.  Should we either rename this to mDelayConfirmCurrentImageSec or have mDelayNotifyUpdateAppliedSeconds be applied directly to the delay of calling NotifyUpdateApplied()?

Reply:
Remaned to mDelayConfirmCurrentImageSec.